scripts/tools/extract-tools-github.js

```diff
@@ -15,8 +15,7 @@ const getData = async () => {
     
     return result.data;
   } catch (err) {
-    console.log(err);
-    throw err;
+    throw new Error(err)
```

Comment:
You can throw the `err` directly over here, like `throw err`. Only use `new Error()` thing when you have to pass a custom string along with the error message.